### PR TITLE
Logging documentation

### PR DESCRIPTION
**Pull request by chrisdunelm**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -93,12 +93,31 @@ Querying data:
 
 # Google.Logging.V2 (alpha)
 
-A library for working with [Stackdriver Logging](https://cloud.google.com/logging/).
+A library for working with [Google Stackdriver Logging](https://cloud.google.com/logging/).
+
+## Log4net integration - Google.Logging.Log4Net (NuGet) (alpha)
```
Comment (jskeet) [inline docs/index.md]: The NuGet part should be a link. For example:

```text
# Google.Pubsub.V1 ([NuGet](https://www.nuget.org/packages/Google.Pubsub.V1)) (beta)
```

Ditto below.

File: docs/index.md [hunk 2]
```diff
@@ -93,12 +93,31 @@ Querying data:
...
+
+## Basic logging library (NuGet) (alpha)
 
 (Depends on `Google.Logging.Type` for version-agnostic types.)
 
-See [LoggingServiceV2Client](obj/api/Google.Logging.V2.LoggingServiceV2Client.yml)
-for details.
+If required,
+[LoggingServiceV2Client](obj/api/Google.Logging.V2.LoggingServiceV2Client.yml)
+provides a low-level abstraction over the raw RPC API
```
Comment (jskeet) [inline docs/index.md]: Full stop at end of sentence.

File: docs/index.md (hunk 1 above)
Comment (chrisdunelm) [inline docs/index.md]: There isn't a NuGet package yet, so this is a placeholder.
I'll remove it for now.

File: docs/index.md (hunk 2 above)
Comment (chrisdunelm) [inline docs/index.md]: Done

**Comment by chrisdunelm** [comment]
PTAL

File: docs/index.md (hunk 1 above)
Comment (jskeet) [inline docs/index.md]: Righto. I'll revise all of these bits when everything is in beta and on nuget.org anyway.

**Comment by jskeet** [comment]
LGTM. Happy for you to merge this before the actual snippet if you want, just to get it out of the way - I don't think the fact that the snippets are broken for a couple of hours will bother anyone.

**Pull request merged**